src/nvim/window.c

```diff
@@ -633,6 +633,16 @@ void win_set_minimal_style(win_T *wp)
     xfree(wp->w_p_scl);
     wp->w_p_scl = (char_u *)xstrdup("auto");
   }
+
+  // colorcolumn: cleared
+  if (wp->w_p_cc) {
+    xfree(wp->w_p_cc);
+    wp->w_p_cc = NULL;
+  }
+  if (wp->w_p_cc_cols) {
```

Comment:
This is not needed. Window options are always interpreted after this function is called.

Reply:
Ok, I assume that to mean it’s a noop?
So is it not possible to do what I want to do which is to clear out cc for the minimal style?
Or do I shift the logic somewhere else?

Reply:
I did not say anything about it being impossible. You clear out `wp->w_p_cc` above, that is all that is needed. We already have code that translates from `w_p_cc` to `w_p_cc_cols`, this need not be copied and pasted to every specific place `w_p_cc` is being changed. This is just like the other string options this function already changes.